src/main/java/net/honux/chess/Board.java

```diff
@@ -3,20 +3,82 @@
 import net.honux.chess.pieces.Pawn;
 
 import java.util.ArrayList;
+import java.util.List;
 
 public class Board {
 
-    ArrayList<Pawn> pawns = new ArrayList<Pawn>();
+    List<Pawn> whitePawns = new ArrayList<Pawn>();
+    List<Pawn> blackPawns = new ArrayList<Pawn>();
 
-    public void add(Pawn pawn) {
-        pawns.add(pawn);
+
+    public void whitePawnsAdd(Pawn pawn){
+        whitePawns.add(pawn);
+    }
+
+    public void blackPawnsAdd(Pawn pawn){
+        blackPawns.add(pawn);
     }
 
     public int size() {
-        return pawns.size();
+        return whitePawns.size()+blackPawns.size();
+    }
+
+    public Pawn findWhitePawn(int index) {return whitePawns.get(index);}
+
+    public Pawn findBlackPawn(int index) {
+        return blackPawns.get(index);
     }
 
-    public Pawn findPawn(int index) {
-        return pawns.get(index);
+    public void initialize() {
+        int boardSize = 8;
+
+        for (int i = 1; i <= boardSize; i++) {
+            whitePawnsAdd(new Pawn(Pawn.WHITE_COLOR,Pawn.WHITE_REPRESENTATION));
+            blackPawnsAdd(new Pawn(Pawn.BLACK_COLOR,Pawn.BLACK_REPRESENTATION));
+        }
+
     }
+
+    public String getWhitePawnsResult() {
+        StringBuilder whiteResult = new StringBuilder();
+
+        for (Pawn whitePawn : whitePawns){
+            whiteResult.append(whitePawn.getRepresentation());
+        }
+        return whiteResult.toString();
+
+    }
+
+    public String getBlackPawnsResult() {
+        StringBuilder blackResult = new StringBuilder();
+
+        for (Pawn blackPawn : blackPawns){
+            blackResult.append(blackPawn.getRepresentation());
+        }
+        return blackResult.toString();
+    }
+
+    public String print(){
+        StringBuilder printResult = new StringBuilder();
+        String line = "........";
+        char lineBreak = '\n';
+        printResult.append(line);
+        printResult.append(lineBreak);
+        printResult.append(getBlackPawnsResult());
+        printResult.append(lineBreak);
+        printResult.append(line);
+        printResult.append(lineBreak);
+        printResult.append(line);
+        printResult.append(lineBreak);
+        printResult.append(line);
+        printResult.append(lineBreak);
+        printResult.append(line);
+        printResult.append(lineBreak);
+        printResult.append(getWhitePawnsResult());
+        printResult.append(lineBreak);
+        printResult.append(line);
+        return printResult.toString();
```

Comment:
```suggestion
        printResult.append(line).append(lineBreak);
        printResult.append(getBlackPawnsResult()).append(lineBreak);
        printResult.append(line).append(lineBreak);
        printResult.append(line).append(lineBreak);
        printResult.append(line).append(lineBreak);
        printResult.append(line).append(lineBreak);
        printResult.append(getWhitePawnsResult()).append(lineBreak);
        printResult.append(line).append(lineBreak);
        return printResult.toString();
```
이렇게 붙여줘도 되고, printResult를 아예 없애줘도 좋을 것 같아요.

Reply:
이렇게 라인을 줄인 이유는 세로로 너무 길어도 가독성이 떨어지기 떄문이에요.

Reply:
emptyLine을 메소드로 빼고, 줄바꿈을 다른 OS에서도 사용가능 할 수 있도록` System.getProperty("line.separator");` 로 변경했습니다.
디온이 말씀주신것과 같이 변경해보았는데, 확실히 전보다는 가독성이 좋아진 것 같지만, 아직도 가독성이 그리 좋지 않은 것 같아서,.. 어떻게 변경해야 더 눈에 확 잘 들어올지 고민이 되네요

Reply:
변경후

``` java
 private String emptyLine(){
        String emptyLine = "........";
        return emptyLine;
    }

    public String boardPrint() {
        StringBuilder printResult = new StringBuilder();
        String lineBreak = System.getProperty("line.separator");
        printResult.append(emptyLine()).append(lineBreak);
        printResult.append(getBlackPawnsResult()).append(lineBreak);
        printResult.append(emptyLine()).append(lineBreak);
        printResult.append(emptyLine()).append(lineBreak);
        printResult.append(emptyLine()).append(lineBreak);
        printResult.append(emptyLine()).append(lineBreak);
        printResult.append(getWhitePawnsResult()).append(lineBreak);
        printResult.append(emptyLine());
        return printResult.toString();
    }
```

Reply:
정답이 없는 문제라 ㅎㅎ emptyLine에 lineBreak를 더해서 리턴해줘도 좋겠네요!